include/swift/AST/PrintOptions.h

```diff
@@ -425,6 +425,7 @@ struct PrintOptions {
     result.AbstractAccessors = false;
     result.PrintForSIL = true;
     result.PrintInSILBody = true;
+    result.PreferTypeRepr = false;
```

Comment:
SIL only really cares about the canonical types of things, so ignoring the type repr makes sense to me.